tools/world-builder/projects.json

```diff
@@ -13,7 +13,7 @@
   },
   {
     "git": "https://github.com/mongodb/docs.git",
-    "branches": ["v3.0", "v3.2", "v3.4", "master"],
+    "branches": ["v3.0", "v3.2", "v3.4", "v3.6", "master"],
```

Comment:
@i80and  -- now that for many of our Makefile has it so that on make master, we delete the build/public directory, does the ordering of the branches matter?

Reply:
Order doesn't matter; each branch gets its own clone. However, it *is* true that the world builder doesn't create symlinks.

This is a pretty big gotcha right now.